pandas/tests/frame/test_cumulative.py

```diff
@@ -0,0 +1,120 @@
+"""
+Tests for DataFrame cumulative operations
+
+See also
+--------
+tests.series.test_cumulative
+"""
+
+import numpy as np
+
+from pandas import DataFrame, Series
+import pandas.util.testing as tm
+
+
+class TestDataFrameCumulativeOps:
```

Comment:
Not a huge deal but probably don't even need the classes now

Reply:
for whatever reason i still find this style more comfortable, but agreed on the "not a huge deal"